src/ImageIO/CGImageDestination.cs

```diff
@@ -502,8 +502,9 @@ public bool CopyImageSource (CGImageSource image, CGImageDestinationOptions opti
 
 		public void AddAuxiliaryDataInfo (CGImageDestination dest, CGImageAuxiliaryDataType auxiliaryImageDataType, CGImageAuxiliaryDataInfo auxiliaryDataInfo)
 		{
-			var dict = auxiliaryDataInfo == null ? null : auxiliaryDataInfo.ToDictionary ();
-			CGImageDestinationAddAuxiliaryDataInfo (dest.Handle, auxiliaryImageDataType.GetConstant ().Handle, dict == null ? IntPtr.Zero : dict.Handle);
+			using (var dict = auxiliaryDataInfo?.ToDictionary ()) {
+				CGImageDestinationAddAuxiliaryDataInfo (NativeObjectHelper.GetHandle (dest), auxiliaryImageDataType.GetConstant ().Handle, dict == null ? IntPtr.Zero : dict.Handle);
```

Comment:
`NativeObjectHelper.GetHandle` is an extension method so you can do:

```
CGImageDestinationAddAuxiliaryDataInfo (dest.GetHandle (), auxiliaryImageDataType.GetConstant ().GetHandle (), dict.GetHandle ());
```
it's shorter and also cover the case where a constant might be `null` (e.g. if only available in a more recent version of iOS/macOS)